### DISCUSSION TITLE
Big install size on Windows 64bit

### DISCUSSION THREAD

**Discussion by marcelflymark** [Tribler/tribler#8166] [General]
Hi Tribler people,

I just installed Tribler 7.14.0 stable on Windows 10 64bit.

My Question: It takes up 775 MB. Is this the development tree by any chance?

For now I uninstalled it.

Best regards,

Marcel

**Comment by qstokkink** [comment]
Hi Marcel. Thanks for trying out Tribler. Over the course of the Tribler 7 series, our software has become a bit.. corpulent. 

In the upcoming Tribler 8.0 release, we have severely cut down on our size. Our current bleeding-edge build clocks in at `44.9 MB`. We are still in the process of quality control but we hope to release this for you soon!
